templates/30-validation-webhook.servicemonitor.yaml.tmpl

```diff
@@ -0,0 +1,17 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    name: "#SVCNAME#"
+  name: "#SVCNAME#"
+  namespace: "#NAMESPACE#"
+spec:
+  endpoints:
+  - port: "https"
+    scheme: "https"
+    tlsConfig: 
+      insecureSkipVerify: true
```

Comment:
We'd need to replace the insecure bit with the actual TLS config before we go into production with that.

Reply:
@rogbas when you say production does that mean both environments (stage and prod) or just prod?